.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
This is what actually enforces the checks or else we just get a warning and not `enforced errors`